languagetool-language-modules/en/src/main/resources/org/languagetool/resource/en/hunspell/spelling.txt

```diff
@@ -3371,4 +3371,32 @@ GDPR
 iKONIK
 Camaro
 Camila
-GmbH
\ No newline at end of file
+GmbH
+underreport
+underreported
+underreports
+underreporting
+Salzburg
+Marcin
+Michal
```

Comment:
Very close to "Michael" and probably much less common, I'd says leave it out because of that.

Reply:
I just noticed that it's "Michał" (notice the special 'ł'). I changed it to that.